packages/design-system-tokens/src/lib/exportScss.ts

```diff
@@ -18,32 +27,26 @@ export const exportScss = (fileDescriptors: FileDescriptor[], outPath: string):
      */
     // eslint-disable-next-line @typescript-eslint/no-var-requires
     const importedModule = require(`${file.moduleImportName}`);
+    const filename = `${outPath}/${file.exportFileName}.scss`;
+    let output = '';
+    let type = 'tokens';
```

Comment:
It looks like this could maybe be replaced with a boolean to simplify, but I really like that you pulled this out into a variable so it's clear when we get to the if/else what is happening.

Reply:
Oh, but now I see that the string helps us understand what the other case is. It's a token or a theme, and a boolean name would only capture one of those cases.

Reply:
Could maybe consolidate this to
```suggestion
    const type = importedModule.default.description ? 'theme' : 'tokens';
```
so we can see what the two possible values are right away.

Reply:
Sorry, this is a nit-pick